Modules/Module_Processing/Module_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -76,36 +76,32 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
         nFields = length(jsonFields);
         for iField=1:nFields
             if ~isempty(Parms.(jsonFields{iField}))
+				% Saves the new field for further testing and formatting
+				newField = Parms.(jsonFields{iField});
+
+				if iscell(newField)
+					% Verify that all gathered cells include strings and not cells inside
```

Comment:
I think we do this already with xASL_num2str below, but this probably doesn't hurt?
Or is xASL_num2str better because it can contain numbers which would not correctly be converted to strings here?

Reply:
This is actually not about numbers but about strings. So a DICOM field having a couple of strings, which then gets converted to a cell array and we only need to concatenate to strings again :)